danger-kotlin-library/src/main/kotlin/com/danger/dangerkotlin/DangerResults.kt

```diff
@@ -30,7 +31,7 @@ internal data class DangerResults(var fails: Array<Violation>,
     }
 }
 
-internal class Meta {
-    val runtimeName = "Danger Kotlin"
-    val runtimeHref = "https://danger.systems"
-}
\ No newline at end of file
+internal data class Meta(
+    val runtimeName: String = "Danger Kotlin",
```

Comment:
not need to specify the type here, but if you want to keep it is up to you 😄 minor

Reply:
The compiler required it :(

```
A type annotation is required on value parameter
```